modules/ROOT/pages/virtual-private-cloud.adoc

```diff
@@ -13,7 +13,7 @@ image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoin
 
 The Anypoint Virtual Private Cloud (VPC) offering allows you to create a virtual, private, and isolated network segment in the cloud to host your CloudHub workers.
 
-Connecting to your Anypoint VPC extends your corporate network and allows CloudHub workers to access resources behind your corporate firewall. You can connect on-premises data centers through a secured VPN tunnel, or a private AWS VPC through VPC peering, or by using AWS Direct Connect.
+Connecting to your Anypoint VPC extends your corporate network and allows CloudHub workers to access resources behind your corporate firewall. You can connect on-premises data centers through a secured VPN tunnel or transit gateway attachment, a private AWS VPC through VPC peering, or by using AWS Direct Connect.
```

Comment:
The three list items in the second sentence don't seem parallel:

connect centers through a tunnel or attachment
(connect?) VPC through peering
(connect? what?) by using